### PR TITLE
Add ability to downgrade admin to player

### PR DESCRIPTION
**Pull request by andrrac**
Implementation for issue #324

### CODE REVIEW

File: src/main/java/servlets/admin/userManagement/DowngradeAdmin.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+ * The Security Shepherd project is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.<br/>
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with the Security Shepherd project.  If not, see <http://www.gnu.org/licenses/>. 
+ *
+ */
+public class DowngradeAdmin extends HttpServlet
```
Comment (markdenihan) [inline src/main/java/servlets/admin/userManagement/DowngradeAdmin.java]: This Servlet needs a Junit Test. You can use the examples from the other Admin Servlet JUnit Tests

File: src/test/java/dbProcs/GetterTest.java [hunk 2]
```diff
@@ -3144,6 +3144,7 @@ public void testFindAdminById()
 	
 	@Test
 	public void testGetAdmins()
+	//TODO
```
Comment (markdenihan) [inline src/test/java/dbProcs/GetterTest.java]: Is there somthing you were planning to add here?

File: src/main/java/servlets/admin/userManagement/DowngradeAdmin.java (hunk 1 above)
Comment (andrrac) [reply src/main/java/servlets/admin/userManagement/DowngradeAdmin.java]: Added DowngradeAdminsTest

File: src/test/java/dbProcs/GetterTest.java (hunk 2 above)
Comment (andrrac) [reply src/test/java/dbProcs/GetterTest.java]: Missed it. I've removed it

**Comment by andrrac** [comment]
Should be good now

**Pull request merged**